### ISSUE TITLE
Keyboard navigation between different files is broken

### ISSUE THREAD

**Issue by Discookie** [Ericsson/CodecheckerVSCodePlugin#71]
Keyboard navigation between repr. steps doesn't work outside of the active report's file, so when it would jump into a different file, it doesn't open that file. Same thing when jumping with the cursor in a different file.
